VolumeLogic/VolumeMixer.py

```diff
@@ -0,0 +1,45 @@
+from VolumeLogic import ChangedValue, IObserver, Subject
+
+
+class VolumeMixer(IObserver, Subject):
+    """Collection of Applications whose volume is being controlled."""
+
+    actions = {
+        ChangedValue.VOLUME: lambda application, value: application.volume(value),
+        ChangedValue.COLOR: lambda application, value: application.color_matrix(value),
+    }
+
+    def __init__(self):
+        super().__init__()
+        self.applications = [None] * 5
```

Comment:
Can we get rid of the magic number here? set a global 5 which a descriptive name and use that.